Jenkinsfile

```diff
@@ -120,6 +138,12 @@ podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultCon
             // https://stackoverflow.com/questions/1636889
             if (!devel) {
                 utils.rsync_out("builds", "builds")
+                if (utils.path_exists("/.aws")) {
```

Comment:
any preference on just using ENV vars here instead of a config file there?

Reply:
Heh, I explicitly chose to move away from env vars actually to raise the barrier a bit more on how easily credentials could leak e.g. into logs.

Reply:
Side note, for AWS it's best to use https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_use-resources.html if you can.

Reply:
> Heh, I explicitly chose to move away from env vars actually to raise the barrier a bit more on how easily credentials could leak e.g. into logs.

yeah. i've just always done it the other way so I have a slight preference. We can stick with this.

Reply:
> Side note, for AWS it's best to use https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_use-resources.html if you can.

Hmm neat, hadn't seen this before. I don't want to address this right now in this patch, but I don't want to lose track of it either, so filed https://github.com/coreos/fedora-coreos-pipeline/issues/63.